src/main/java/com/manardenza/TestUtils.java

```diff
@@ -20,10 +21,52 @@
     public static final Date RESERVED_TO = new Date(117, 1, 15);
     public static final User USER = new User(FIRST_NAME, LAST_NAME);
     public static final Room ROOM = new Room("Room", 2, 300);
-    public static final List<Room> ROOMS_LIST = new ArrayList<>();
-    public static final Hotel HOTEL = new Hotel(HOTEL_NAME, CITY, ROOMS_LIST);
+    public static final Hotel HOTEL = new Hotel(HOTEL_NAME, CITY, getRoomsSeedList());
     public static final Map<String, List<Room>> ROOMS_MAP = new HashMap<>();
 
-    private TestUtils() {}
+    public static List<Room> getRoomsSeedList() {
+        List<Room> testRoomsList = new ArrayList<>();
+        testRoomsList.add(new Room("DOUBLE ROOM", 2, 300));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "DOUBLE ROOM" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)